torch/_inductor/mkldnn.py

```diff
@@ -524,8 +526,21 @@ def create_unary_module(node: torch.fx.node):
         F.gelu: nn.GELU,
         F.relu6: nn.ReLU6,
         F.silu: nn.SiLU,
+        torch.relu: nn.ReLU,
+        torch.sigmoid: nn.Sigmoid,
+        torch.tanh: nn.Tanh,
+        torch.Tensor.relu: nn.ReLU,
+        torch.Tensor.sigmoid: nn.Sigmoid,
+        torch.Tensor.tanh: nn.Tanh,
```

Comment:
can we use "relu", "sigmoid" and "tahn" directly as keys for tensor methods?

Reply:
Good point. Changed.